symtabAPI/src/Object.h

```diff
@@ -178,6 +178,8 @@ class Object : public boost::basic_lockable_adapter<boost::mutex>
     virtual Offset getTOCoffset(Offset) const { return 0; }
     virtual void setTOCoffset(Offset) { }
 
+    virtual bool riscvUsesRVC() { return false; }
```

Comment:
Having architecture-specific interfaces at the ObjectElf` level I can mostly understand, but I'm curious why this is at the `Object` level? This is supposed to be a generic interface into a file blob. Admittedly, though, it is very centered on elf/dwarf.

Reply:
Ah, ok. Let's rename it to something like `usesCompressedInstructionFormat`?

>  RISC-V PE
> Although Dyninst does not currently support PE

Let's not think about that...

Reply:
OK. Renamed to `usesCompressedInstructionFormat`